.github/workflows/ci-windows.yaml

```diff
@@ -19,8 +19,8 @@ jobs:
         with:
           miniconda-version: "latest"
           python-version: ${{ matrix.python-version }}
-          environment-file: continuous_integration/environment.yml
-          activate-environment: testenv
+          environment-file: continuous_integration/environment-windows.yml
+          activate-environment: distributed
```

Comment:
Note that we try to avoid using the name `distributed` in isolation.  It's a bit overly generic.  I recommend `dask-distributed` instead.

Reply:
fixed